CMakeLists.txt

```diff
@@ -5,6 +5,9 @@ project(TheoIDE VERSION 0.1.0 LANGUAGES CXX)
 set(QT_VERSION 6.7)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CPACK_PACKAGE_VERSION ${PROJECT_VERSION})
+set(CPACK_PACKAGE_NAME TheoIDEApp-Setup)
+set(CPACK_DEBIAN_PACKAGE_DEPENDS "libxcb-cursor-dev")
```

Comment:
Does it work with `libxcb-cursor0` too, because `-dev` packages include headers which should not be required at runtime.

Reply:
Yes that also works